inc/functions.inc.php

```diff
@@ -44,6 +44,80 @@ function st_register_widget() {
     }
 }
 
+function tp_disp_boolean($bool_text)
+{
+    $bool_text = (string)$bool_text;
+    if (empty($bool_text) || '0' === $bool_text || 'false' === $bool_text) {
+        return 'false';
+    }
+
+    return 'true';
+}
+
+/**
+ * Add filter for custom taxonomies on edit posts page
+ * 
+ */
+
+add_action('restrict_manage_posts', 'taxopress_add_posts_filter');
+add_filter( 'parse_query', 'taxopress_user_filters');
+
+function taxopress_add_posts_filter() {
+
+    global $wpdb, $wp_query, $table_prefix;
+    
+    $post_type = (isset($_GET['post_type'])) ? $_GET['post_type'] : 'post';
+    
+    if ($post_type == 'post'){ //only add filter to posts
```

Comment:
I already replied your comment on the issue page.

The dropdown should be enabled on the taxonomies post types